feature-libs/cart/quick-order/components/container/quick-order-container.component.ts

```diff
@@ -1,16 +1,36 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { ActiveCartService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { QuickOrderStatePersistenceService } from '../../core/services/quick-order-state-persistance.service';
+import { QuickOrderService } from '../../core/services/quick-order.service';
 
 @Component({
   selector: 'cx-quick-order-container',
   templateUrl: './quick-order-container.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class QuickOrderComponent {
+export class QuickOrderComponent implements OnInit {
+  cartId$: Observable<string> = this.activeCartService.getActiveCartId();
```

Comment:
Those two assignments should be in the ngOnInit, only declarations should be here.